Emby.Server.Implementations/LiveTv/Listings/SchedulesDirect.cs

```diff
@@ -812,10 +811,10 @@ public async Task<List<ChannelInfo>> GetChannels(ListingsProviderInfo info, Canc
                     channelInfo.ImageUrl = station.logo.URL;
                 }
 
-                array[i] = channelInfo;
```

Comment:
I think this is the only line that needed to be reverted?

Reply:
That line could have been changed to array.Append(channelInfo), yes, but with that change there's no reason to use a regular for loop instead of a foreach loop.